lib/mandrill_dm/message.rb

```diff
@@ -276,6 +276,15 @@ def inline_attachments?
       mail.attachments.any?(&:inline?)
     end
 
+    def html_content?
+      return true if mail.content_type.nil?
+      mail.content_type =~ %r{text/html} ? true : false
```

Comment:
@spovich @Sysqa content type validation should be strict, not anything that contains `text/html`